src/main/java/com/powsybl/openloadflow/ac/outerloop/AcLoadFlowParameters.java

```diff
@@ -38,10 +38,12 @@
 
     private final boolean minImpedance;
 
+    private final boolean specificCompatibility;
```

Comment:
Now that we have switched to powsybl 3.4 we can rename it twtSplitShuntAdmittance?

Reply:
Yes, you're right.